modules/elasticache/locals.tf

```diff
@@ -0,0 +1,13 @@
+locals {
+  namespace = "devops-ic-elasticache"
+
+  engine                      = "redis"
+  engine_version              = "6.x"
+  parameter_group_name        = "default.redis6.x"
+  node_type                   = "cache.t2.micro"
```

Comment:
shouldn't this node_type be controlled by a variable, so that when calling this module from `core/main.tf`, we can specify a different value for each environment?

Reply:
I got the idea 🎉 , it was updated in 5e37f4e ✅